packages/contact-center/store/src/storeEventsWrapper.ts

```
@@ -0,0 +1,197 @@
+  }
+
+  get showMultipleLoginAlert() {
+    return this.store.showMultipleLoginAlert;
+  }
+
+  get currentTheme() {
+    return this.store.currentTheme;
+  }
+
+  setCurrentTheme(theme: string): void {
+    return this.store.setCurrentTheme(theme);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)